### PR TITLE
DOC: Clarifies ANCOM usage in Moving Pictures tutorial

### LINKED ISSUE #125: ANCOM Clarification
The `Moving Pictures` [tutorial](https://docs.qiime2.org/2017.2/tutorials/moving-pictures/#differential-abundance-analysis) says the following about `ancom`:

> ANCOM operates on a `FeatureTable[Composition]` QIIME 2 artifact, which is based on relative frequencies of features on a per-sample basis, but cannot tolerate frequencies of zero. We work around this by adding a pseudocount of 1 to every count in our `FeatureTable[Frequency]` table. 

The prose indicates that we are working with relative frequencies, but the registered input type to the `add-psuedocount` method is `FeatureTable[Frequency]`. Looking at the [plugin](https://github.com/qiime2/q2-composition/blob/master/q2_composition/_impute.py), it doesn't look like we are converting from `Frequency` to `RelativeFrequency` anywhere. Am I misunderstanding something? Do we need to update this prose to clarify something?

### PR DESCRIPTION
**Pull request by Kleptobismol**
Fixes some tricky wording regarding the use of ANCOM

Fixes #125

### CODE REVIEW

File: source/tutorials/moving-pictures.rst [hunk 1]
```diff
@@ -367,7 +367,8 @@ Next, we can view the taxonomic composition of our samples with interactive bar
...
+Finally, we can quantify the process of identifying taxa that are differentially abundance (or present in different abundances) across sample groups. We do that using ANCOM (`Mandal et al. (2015)`_), which is implemented in the ``q2-composition`` plugin. ANCOM operates on a ``FeatureTable[Composition]`` QIIME 2 artifact, which is based on relative frequencies of features on a per-sample basis, but cannot tolerate frequencies of zero. In order to
+use ANCOM, a ``FeatureTable[Composition]`` artifact must be built using frequency data. To do this, a ``FeatureTable[Frequency]`` artifact containing the relative frequencies must be used as input to add-pseudocount, which will then output the ``FeatureTable[Composition]`` artifact necessary to use ANCOM. We can run this on the ``BodySite`` category to determine what features differ in abundance across body sites. This step may take about 5 minutes to complete.
```
Comment (thermokarst) [inline source/tutorials/moving-pictures.rst]: Can you drop this line break?

Comment (Kleptobismol) [inline source/tutorials/moving-pictures.rst]: done

File: source/tutorials/moving-pictures.rst [hunk 2]
```diff
@@ -367,7 +367,7 @@ Next, we can view the taxonomic composition of our samples with interactive bar
...
+Finally, we can quantify the process of identifying taxa that are differentially abundance (or present in different abundances) across sample groups. We do that using ANCOM (`Mandal et al. (2015)`_), which is implemented in the ``q2-composition`` plugin. ANCOM operates on a ``FeatureTable[Composition]`` QIIME 2 artifact, which is based on relative frequencies of features on a per-sample basis, but cannot tolerate frequencies of zero. In order to use ANCOM, a ``FeatureTable[Composition]``artifact must be built using frequency data. To do this, a ``FeatureTable[Frequency]`` artifact containing the relative frequencies must be used as input to add-pseudocount, which will then output the ``FeatureTable[Composition]`` artifact necessary to use ANCOM. We can run this on the ``BodySite`` category to determine what features differ in abundance across body sites. This step may take about 5 minutes to complete.
```
Comment (ebolyen) [inline source/tutorials/moving-pictures.rst]: `...``FeatureTable[Composition]``artifact must be b...` -> there's a missing space between the backtick and artifact which will cause issues.

`FeatureTable[Composition]` is not strictly based on relative frequencies. Relative frequencies are also not the input to `add-pseudocount`.

We can brainstorm on some better language in-person if you'd like.

File: source/tutorials/moving-pictures.rst [hunk 3]
```diff
@@ -367,8 +367,7 @@ Next, we can view the taxonomic composition of our samples with interactive bar
...
-
+Finally, we can quantify the process of identifying taxa that are differentially abundance (or present in different abundances) across sample groups. We do that using ANCOM (`Mandal et al. (2015)`_), which is implemented in the ``q2-composition`` plugin. ANCOM operates on a ``FeatureTable[Composition]`` QIIME 2 artifact, which is based on frequencies of features on a per-sample basis, but cannot tolerate frequencies of zero. To build the composition artifact, a ``FeatureTable[Frequency]``  artifact must be provided to add-pseudocount (an imputation method), which will then produce the ``FeatureTable[Composition]`` artifact. We can then run ANCOM on the ``BodySite`` category to determine what features differ in abundance across body sites. This step may take about 5 minutes to complete.
```
Comment (ebolyen) [inline source/tutorials/moving-pictures.rst]: Sorry just noticed a typo: in the first sentence, `differentially abundance` should be `differentially abundant`

Also I would drop the `then` from this fragment: `...(an imputation method), which will <then> produce...`

Comment (Kleptobismol) [inline source/tutorials/moving-pictures.rst]: done

**Comment by ebolyen** [comment]
Thanks @Kleptobismol

**Pull request merged**